### ISSUE TITLE
Display back-end errors in console of browser

### ISSUE THREAD

**Issue by lbeaufort** [fecgov/fecfile-web-app#1869]
We shouldn't throw popups for server errors.

[!https://github.com/fecgov/fecfile-web-app/assets/31420082/e953d7b5-8799-470b-9b4d-c45f608098b8!](https://github.com/fecgov/fecfile-web-app/assets/31420082/e953d7b5-8799-470b-9b4d-c45f608098b8)

### QA Notes ###

 
 A screenshot of a back-end error message in the console will be put in the ticket for verification.

### DEV Notes ###
Error messages other than 403 from the back-end should be sent to the browser console.
Put a screenshot of a console error message in the comments of the ticket for QA.

### Design ###

 
 null

 See full ticket and images here: [FECFILE-429](https://fecgov.atlassian.net/browse/FECFILE-429)

**Comment by AureliaKhorsand** [comment]
@MitchellTCG what is this on hold for? Who is this waiting on? If I can help get more information for it, let me know.

**Comment by JonellaCulmer** [comment]
@lbeaufort Want to get additional clarity on the scope of this. We currently handle 500 server errors on .gov with a page. Should 400 errors be treated differently and just appear in the console? 400 errors look like access issues but I'm concerned about developing processes for something that should potentially be handled on the login.gov side. Happy to discuss at your convenience.

**Comment by lbeaufort** [comment]
We've had some back and forth in Slack, but adding here for posterity that this is the only `alert`/pop-up in the code (line 45):
https://github.com/fecgov/fecfile-web-app/blob/0f8172530677de9f7685f74982c19f7b4ab49bfb/front-end/src/app/shared/interceptors/http-error.interceptor.ts#L28-L48
